project.yaml

```diff
@@ -2115,12 +2117,13 @@ Layer:
             name,
             operator,
             ref,
+            ST_Length(way) AS line_length,
```

Comment:
Don't use line length, use line length relative to zoom, or what @imagico indicated earlier. For better non-mercator compatibility, use `ST_Length(way)/NULLIF(SQRT(!pixel_width!::real*!pixel_width!::real),0)`